torch/csrc/stable/library.h

```diff
@@ -88,7 +88,7 @@ struct FromImpl<std::nullopt_t> {
 // std::optional<T> or a std::nullopt.
 template <typename T>
 struct FromImpl<std::optional<T>> {
-  static StableIValue call(std::optional<T> val) {
+  static StableIValue call(const std::optional<T>& val) {
```

Comment:
if T can be something other than a cheaply copyable primitive then fine, otherwise you should revert this one too.